readme.md

```diff
@@ -98,8 +98,8 @@ Whether to maintain image aspect ratio or not.
 
 **Only works for `terminalImage.gifBuffer` or `terminalImage.gifFile`**
 
-Type: `number`\
-Default: `Infinity`
+Type: `number | Infinity`\
```

Comment:
`Infinity` is a number.

Reply:
https://github.com/sindresorhus/round-to#precision

Reply:
That is different. There we support only Integer or Infinity. Here, we support any number.

Reply:
Fixed!